Cannot find module @actions/exec

**Description**
Error occurs when `npm run dev` is run

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'express-autodocs'
2. `npm install`
3. `npm run dev`
4. `Error:  Cannot find module 'express-autodocs\node_modules\@actions\exec\lib\exec.js'. Please verify that the package.json has a valid "main" entry`

**Expected behavior**
Docs should be generated in output folder

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop:**
 - OS: Windows 10


**Additional context**
The error is caused due to first line in [helpers/templateHelpers/createDocs.js](https://github.com/Pika1998/express-autodocs/blob/master/helpers/templateHelpers/createDocs.js) which isn't used anyway

The error is solved by removing this line

--- a/helpers/templateHelpers/createDocs.js
+++ b/helpers/templateHelpers/createDocs.js
@@ -1,17 +1,16 @@
-const exec = require("@actions/exec");
-const generateMarkdowndocs = require("./markdown/generateMarkdownDocs");
-const generateStandardDocs = require("./standard/generateStandardDocs");
+const generateMarkdowndocs = require('./markdown/generateMarkdownDocs');
+const generateStandardDocs = require('./standard/generateStandardDocs');
 
 function createDocs(apis, outputFormat, outputBranch, token, docsTitle) {
   switch (outputFormat) {
-    case "markdown":
+    case 'markdown':
       generateMarkdowndocs(apis, outputBranch, token, docsTitle);
       break;
-    case "standard":
+    case 'standard':
       generateStandardDocs(apis, outputBranch, token, docsTitle);
       break;
     default:
-      console.log("\x1b[1m", "\x1b[31m", `❌ Invalid Output Format`, "\x1b[0m");
+      console.log('\x1b[1m', '\x1b[31m', `❌ Invalid Output Format`, '\x1b[0m');
       return;
   }
 }

--- a/package.json
+++ b/package.json
@@ -25,7 +25,6 @@
   "homepage": "https://github.com/Pika1998/express-autodocs#readme",
   "dependencies": {
     "@actions/core": "^1.2.4",
-    "@actions/exec": "^1.0.4",
     "@actions/github": "^4.0.0",
     "@babel/parser": "^7.11.4",
     "@babel/traverse": "^7.11.0",
@@ -50,6 +49,9 @@
     }
   },
   "lint-staged": {
-    "*.js": ["prettier --write", "git add"]
+    "*.js": [
+      "prettier --write",
+      "git add"
+    ]
   }
 }

